.github/workflows/check.yml

```diff
@@ -6,14 +6,20 @@ on:
   pull_request:
   push:
 
+defaults:
+  run:
+    working-directory: source
+
 jobs:
   run-checks:
     name: Run checks
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why are we fixating this to Ubuntu 20.04 ?  "ubuntu-latest" seems to be less long-term maintenance effort, unless Ubuntu decides to change the package structure.

Reply:
It's to address this: https://github.com/cplusplus/draft/issues/4529#issuecomment-786624331. I point out what's wrong in the second part of its following comment.

Reply:
See the first note at https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on. During the migration period, using `runs-on: ubuntu-latest` could use an Ubuntu 18.04 or 20.04 runner. @jwakely's fork workflow runs (https://github.com/jwakely/draft/actions) were all on Ubuntu 20.04 (for example, see https://github.com/jwakely/draft/runs/1986966401?check_suite_focus=true#step:1:4).

Reply:
Ok, thanks.